starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
+<link rel="stylesheet" href="./css/main.css">
 <head>
   <title>Apple Pie Recipe</title>
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <header class="header">
+    <h1>Apple Pies</h1>
+  </header>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+<section id="intro">
+  <p>This was my grandmother's apple pie recipe. I have never seen another one
+  quite like it. It will always be my favorite and has won me several first place
+  prizes in local competitions. I hope it becomes one of your favorites as well!
+  </p>
+  <center><img class="img_recipeinfo" src="./images/recipe-info.png" alt="recipe-info pic"></center>
+  <hr >
+</section>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+<section id="Ingredients">
+  <h2>Ingredients</h2>
+  <ul>
+    <li>- 1 recipe pastry for a 9 inch double crust pie</li>
+    <li>- 1/2 cup unsalted butter</li>
+    <li>- 3 tablespoons all-purpose flour</li>
+    <li>- 1/4 cup water</li>
+    <li>- 3 tablespoons all-purpose flour</li>
+    <li>- 1/4 cup water</li>
+    <li>- 1/2 cup white sugar</li>
+    <li>- 1/2 cup packed brown sugar</li>
+    <li>- 8 Granny Smith apples - peeled, cored and sliced</li>
+  </ul>
+  <center><img src="./images/cooking-info.png" alt="cooking-info img"></center>
+  <hr>
+</section>
+
+
+<section id="directions">
+  <h2>Directions</h2>
+<p>1) Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
 flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
```

Comment:
Here, instead of creating separate `<p>` tags with "1)", "2)" and "3)", an ordered list would have been a better (and way more scalable) solution.
As a reminder, it works exactly as a `<ul>` but it's written `<ol>`, and instead of bullet points it creates numbers